src/sentry/api/endpoints/organization_member_details.py

```diff
@@ -250,7 +299,23 @@ def put(self, request: Request, organization, member_id) -> Response:
 
         return Response(context)
 
+    @extend_schema(
+        operation_id="Delete a Organization Member",
```

Comment:
I see we have another operation with "Delete an Organization Member". Do we need to differentiate between the two?
https://github.com/getsentry/sentry/blob/1498d05b2f08da36dbf322178799fd0853d38067/src/sentry/scim/endpoints/members.py#L235

Reply:
It might be okay since they're in different tags

```suggestion
        operation_id="Delete an Organization Member",
```

Reply:
@AniketDas-Tekky that endpoint is a part of our SCIM API, which is functionally similar but a different interface, so totally fine to have these both. (end users probably want to use this API instead of SCIM in general, as the SCIM API isn't the most intuitive, and is mainly used by integrations).